server/raft.go

```diff
@@ -1123,6 +1124,15 @@ func (n *raft) loadLastSnapshot() (*snapshot, error) {
 		data:      buf[20+lps : hoff],
 	}
 
+	// We had a bug in 2.9.12 that would allow snapshots on last index of 0.
+	// Detect that here and return err.
+	if snap.lastIndex == 0 {
+		n.warn("Snapshot with last index 0 is invalid, cleaning up")
```

Comment:
On starting my servers I now see pages of this, is it essentially a one off? and I wonder if it really should be a warning since no-one can do anything about it really

Reply:
sounds good to me to change to debug